openwisp_monitoring/monitoring/utils.py

```diff
@@ -58,5 +60,16 @@ def create_database():
     databases = [database['name'] for database in items]
     # if database does not exists, create it
     if settings.INFLUXDB_DATABASE not in databases:
-        db.create_database(settings.INFLUXDB_DATABASE)
-        logger.info('Created inlfuxdb database {0}'.format(settings.INFLUXDB_DATABASE))
+        for n in range(5):
```

Comment:
don't do this here. It should not be the job of this function to retry, this is a lower level function.
Do it in the ready() method of the app, that's where this function will be called.